src/sizes.js

```diff
@@ -3,5 +3,13 @@ export default {
     small: 2,
     big: 4,
     bigger: 6
+  },
+  screenWidth: {
+    xxl: 1800,
```

Comment:
Some set that is quite common.. Bootstrap use different approach, I think because they want to reduce supported widths, so groups are larger.
We don't use any yet, except I need a `xl` this is the only query used and I need it to support (be consistent) with old code.
Other points are optional but as I said quite common. 
Don't think they should be defined once and never change.